src/view/gui/fMainWindow.cpp

```diff
@@ -6093,9 +6111,17 @@ void fMainWindow::ApplicationLIBRASingle()
     return;
   }
 
-  updateProgress(15, "Initializing and running LIBRA compiled by MCC");
+  ApplicationDownloadManager* appDownloadMngr = new ApplicationDownloadManager();
```

Comment:
Same change as stated earlier: let's refrain from using raw pointers unless absolutely needed. If raw pointers are needed, they should be smart pointers.